streams/streams-scala/src/test/resources/log4j.properties

```diff
@@ -0,0 +1,32 @@
+## /*
+##  * Licensed to the Apache Software Foundation (ASF) under one or more
+##  * contributor license agreements. See the NOTICE file distributed with
+##  * this work for additional information regarding copyright ownership.
+##  * The ASF licenses this file to You under the Apache License, Version 2.0
+##  * (the "License"); you may not use this file except in compliance with
+##  * the License. You may obtain a copy of the License at
+##  *
+##  *    http://www.apache.org/licenses/LICENSE-2.0
+##  *
+##  * Unless required by applicable law or agreed to in writing, software
+##  * distributed under the License is distributed on an "AS IS" BASIS,
+##  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+##  * See the License for the specific language governing permissions and
+##  * limitations under the License.
+##  */
+# Set root logger level to DEBUG and its only appender to A1.
+log4j.rootLogger=ERROR, R
+
+# A1 is set to be a ConsoleAppender.
+log4j.appender.A1=org.apache.log4j.ConsoleAppender
+
+log4j.appender.R=org.apache.log4j.RollingFileAppender
+log4j.appender.R.File=logs/kafka-server.log
```

Comment:
The file `logs/kafka-server.log` seems not appropriate as it is not for kafka broker logs right?

Reply:
Will change the name to `kafka-streams-scala.log`